react-components/stories/index.stories.js

```diff
@@ -69,10 +70,10 @@ storiesOf("analysis/view/dialogs", module).add(
     "with test analysis parameters",
     () => <AnalysisParametersDialogTest />
 );
-storiesOf("analysis/view/dialogs", module).add(
-    "with test analysis support",
-    () => <ShareWithSupportDialogTest />
```

Comment:
Was this test intentionally removed?
Its `import` statement was not removed (yet?).

Reply:
Oops 😖